SWLOR.Game.Server/Feature/UsePerkFeat.cs

```diff
@@ -239,8 +239,12 @@ void CompleteActivation(string id, float abilityRecastDelay)
                 DeleteLocalInt(activator, id);
 
                 // Moved during casting or activator died. Cancel the activation.
-                if (GetLocalInt(activator, id) == (int)ActivationStatus.Interrupted || GetCurrentHitPoints(activator) <= 0)
+                if (GetLocalInt(activator, id) == (int)ActivationStatus.Interrupted || GetCurrentHitPoints(activator) <= 0 || GetCurrentAction(activator) != ActionType.Invalid)
+                {
+                    Activity.ClearBusy(activator);
```

Comment:
I don't know if this is possible or not, but clearing the busy state here can affect more than just casting. Can you confirm there's not a way to cancel the busy state on any non-casting actions - for example, harvesting?

Reply:
I'm not sure what the breaking point is. What scenario are we concerned about? Can investigate.